frontend/src/components/TransactionsList.tsx

```diff
@@ -32,9 +68,60 @@ function TransactionsList({
         />
       ) : (
         <>
-          {transactions?.map((tx) => {
-            return <TransactionItem key={tx.paymentHash + tx.type} tx={tx} />;
+          {transactions?.map((tx, i) => {
+            return (
+              <TransactionItem key={tx.paymentHash + tx.type + i} tx={tx} />
```

Comment:
Should we use `updatedAt` in `TransactionItem` component instead of `tx.settledAt || tx.createdAt`